packages/auth/src/api/index.test.ts

```diff
@@ -595,4 +603,34 @@ describe('api/_performApiRequest', () => {
         .and.not.have.property('tenantId');
     });
   });
+
+  context('throws Operation not allowed exception', () => {
```

Comment:
Can we test a happy case for _performRegionalApiRequest to verify it sets the correct request, method, HTTP Headers, etc.